plugins/doc_fragments/tags.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022,  Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+class ModuleDocFragment(object):
+
+    # Minimum requirements for the collection
+    DOCUMENTATION = r'''
+options:
+  tags:
+    description:
+      - A dictionary representing the tags to be applied to the resource.
+      - If the I(tags) parameter is not set then tags will not be modified.
+    type: dict
+    required: false
+    aliases: ['resource_tags']
+  purge_tags:
+    description:
+      - If I(purge_tags=true) existing tags will be purged from the resource to
+        match exactly what is defined by I(tags) parameter.
+      - If the I(tags) parameter is not set then tags will not be modified.
```

Comment:
might be helpful to specifically repeat here that tags will not be modified even if purge_tags=true.